pkg/plugin/sdk/deployment.go

```diff
@@ -455,9 +455,8 @@ type ExecuteStageResponse struct {
 type StageStatus int
 
 const (
-	StageStatusSuccess   StageStatus = 2
-	StageStatusFailure   StageStatus = 3
-	StageStatusCancelled StageStatus = 4
+	StageStatusSuccess StageStatus = 2
+	StageStatusFailure StageStatus = 3
```

Comment:
[Q] I don't remember why we made this enum start from 2; could you teach me? 👀

Reply:
Because these lines are copied from below. It's a bit confusing, so I want to make them start from 1.
https://github.com/pipe-cd/pipecd/blob/ad00a56f0974334cd6f4a0dbca5b505ab19d4f5c/pkg/model/deployment.pb.go#L109-L120

Reply:
I refactored it on this commit.
https://github.com/pipe-cd/pipecd/pull/5597/commits/a755ab8a3859edf1cac522337c67648ca2707ced